spec/index.html

```diff
@@ -299,6 +299,74 @@ <h3>Object Lists</h3>
     Turtle provides a number of ways of writing each.
   </p>
 
+  <section id="sec-version">
+    <h3>Version Announcement</h3>
+
+    <p>The Turtle language has evolved since its origin, and RDF 1.2
+      introduces new features which may confuse non-compliant parsers.
+      RDF 1.2 Turtle introduces the
+      <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a> and
+      <a href="#grammar-production-version"><code>@version</code></a> directives
+      along with an optional `version` <a href="#sec-mediaReg">Media Type parameter</a>.
+      When serializing Turtle with new features such as base direction or triple terms,
+      authors can announce the presense of these features using these directives.
+    </p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="Version announcement">
+      <!--
+      @version "1.2" .
+      @prefix : <http://example.com/> .
```

Comment:
```suggestion
      VERSION "1.2"
      PREFIX : <http://example.com/>
```
(We have been changing examples to use "new" syntax so I think this should go first.

Reply:
This example mirrors the [`@prefix` example](https://www.w3.org/TR/rdf12-turtle/#ex-prefix-decl) in [2.5 IRI](https://www.w3.org/TR/rdf12-turtle/#sec-iri). We should have at least one example using `@version`.